### PR TITLE
Update README to reflect Kubeflow migration to prow build cluster

### LINKED ISSUE #627: Move prow jobs onto the Kubeflow testing cluster
This is issue is tracking moving Kubeflow's prow jobs onto its own build clusters.
https://github.com/kubernetes/test-infra/blob/master/prow/scaling.md#separate-build-clusters

This should allow better isolation and debuggability because we will have access to the prow pod logs.

/assign @clarketm

### PR DESCRIPTION
**Pull request by clarketm**
fix #627

I also updated the string reference of ~gubernator~ to [**spyglass**](https://github.com/kubernetes/test-infra/tree/master/prow/spyglass#spyglass) which is the current artifact viewer in Prow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,22 +240,22 @@ We currently have the following levels of access
...
 
 To access the stackdriver logs 
   
-  * Open stackdriver for project [k8s-prow-builds](https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=k8s-prow-builds&folder&minLogLevel=0&expandAll=false&timestamp=2018-05-22T17:09:26.625000000Z&customFacets&limitCustomFacetWidth=true&dateRangeStart=2018-05-22T11:09:27.032Z&dateRangeEnd=2018-05-22T17:09:27.032Z&interval=PT6H&resource=gce_firewall_rule&scrollTimestamp=2018-05-22T15:40:23.000000000Z&advancedFilter=resource.type%3D"container"%0Aresource.labels.pod_id%3D"15f5a424-5dd6-11e8-826c-0a580a6c0117"%0A)
+  * Open stackdriver for project [k8s-prow-builds](https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=kubeflow-testing&folder&minLogLevel=0&expandAll=false&timestamp=2018-05-22T17:09:26.625000000Z&customFacets&limitCustomFacetWidth=true&dateRangeStart=2018-05-22T11:09:27.032Z&dateRangeEnd=2018-05-22T17:09:27.032Z&interval=PT6H&resource=gce_firewall_rule&scrollTimestamp=2018-05-22T15:40:23.000000000Z&advancedFilter=resource.type%3D"container"%0Aresource.labels.pod_id%3D"15f5a424-5dd6-11e8-826c-0a580a6c0117"%0A)
```
Comment (clarketm) [inline README.md]: Can someone please confirm this updated link is accurate (I doesn't appear that I have the correct perms)?

File: README.md [hunk 2]
```diff
@@ -240,22 +240,22 @@ We currently have the following levels of access
...
 
 To access the stackdriver logs 
   
-  * Open stackdriver for project [k8s-prow-builds](https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=k8s-prow-builds&folder&minLogLevel=0&expandAll=false&timestamp=2018-05-22T17:09:26.625000000Z&customFacets&limitCustomFacetWidth=true&dateRangeStart=2018-05-22T11:09:27.032Z&dateRangeEnd=2018-05-22T17:09:27.032Z&interval=PT6H&resource=gce_firewall_rule&scrollTimestamp=2018-05-22T15:40:23.000000000Z&advancedFilter=resource.type%3D"container"%0Aresource.labels.pod_id%3D"15f5a424-5dd6-11e8-826c-0a580a6c0117"%0A)
+  * Open stackdriver for project [kubeflow-ci](https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=kubeflow-testing&folder&minLogLevel=0&expandAll=false&timestamp=2018-05-22T17:09:26.625000000Z&customFacets&limitCustomFacetWidth=true&dateRangeStart=2018-05-22T11:09:27.032Z&dateRangeEnd=2018-05-22T17:09:27.032Z&interval=PT6H&resource=gce_firewall_rule&scrollTimestamp=2018-05-22T15:40:23.000000000Z&advancedFilter=resource.type%3D"container"%0Aresource.labels.pod_id%3D"15f5a424-5dd6-11e8-826c-0a580a6c0117"%0A)
```
Comment (jlewi) [inline README.md]: The project should be "kubeflow-ci"

Try this
https://console.cloud.google.com/logs/viewer?organizationId=433637338589&project=kubeflow-ci&minLogLevel=0&expandAll=false&customFacets=&limitCustomFacetWidth=true&interval=P7D&resource=k8s_container%2Fcluster_name%2Fkubeflow-testing%2Fnamespace_name%2Ftest-pods&advancedFilter=resource.type%3D%22k8s_container%22%0Aresource.labels.cluster_name%3D%22kubeflow-testing%22%0Aresource.labels.namespace_name%3D%22test-pods%22%0Aresource.labels.pod_name%3D%22bc2f6d5d-7035-11ea-bd6a-f29ce8b0e481%22%0A



I believe you should have access to that project
https://github.com/kubeflow/internal-acls/blob/bbf793c4fdc340341bbda821766b0e676043e7ed/ci-team.members.txt#L2

Comment (clarketm) [reply README.md]: Okay thanks, that link works and has been updated. And does look like I have access.

Comment (clarketm) [reply README.md]: Updated

**Comment by jlewi** [comment]
Thanks. Would you mind updating the section
https://github.com/kubeflow/testing/blob/b8845dd85637b1b437a1717cc24a269407b8cfd1/README.md#logs-and-cluster-access-for-kubeflow-ci

Under k8s-prow-builds it says this is where we run the prow jobs but that is no longer the case.  This should now be under kubeflow-ci

**Comment by clarketm** [comment]
> Thanks. Would you mind updating the section
> https://github.com/kubeflow/testing/blob/b8845dd85637b1b437a1717cc24a269407b8cfd1/README.md#logs-and-cluster-access-for-kubeflow-ci
> 
> Under k8s-prow-builds it says this is where we run the prow jobs but that is no longer the case. This should now be under kubeflow-ci

I missed that, thanks. Updated!

**Comment by jlewi** [comment]
/lgtm
/approve

Thank you

**Pull request merged**